### ISSUE TITLE
test_wcs200 fails

### ISSUE THREAD

**Issue by cehbrecht** [geopython/OWSLib#494]
... it is calling this service:
http://earthserver.pml.ac.uk/rasdaman/ows?service=WCS&request=GetCapabilities&version=2.0.1

and gets the following response:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ows:ExceptionReport version="2.0.1"
    xsi:schemaLocation="http://www.opengis.net/ows/2.0 http://schemas.opengis.net/ows/2.0/owsExceptionReport.xsd"
    xmlns:ows="http://www.opengis.net/ows/2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink">
    <ows:Exception exceptionCode="IOConnectionError">
        <ows:ExceptionText>Cannot open stream to read from URL 'http://earthserver.ecmwf.int/rasdaman/ows/GetBasicCoverageObjects'.</ows:ExceptionText>
    </ows:Exception>
</ows:ExceptionReport>
```

@doclements I think you can help on the service side :)

But in general we can make the test prerequisite check more sophisticated.

**Comment by doclements** [comment]
@cehbrecht it would seem that one of the servers being federated by our is off and the service software isnt smart enough to gracefuly cope with it. I'll work on fixing teh service and also looking for a more stable wcs 2 service to write the tests against.  i'll send a pull request asap

**Comment by cehbrecht** [comment]
@doclements thanks :)

**Comment by cehbrecht** [comment]
Fixed by PR #497.
